src/pages/GoverningBody/AddDocumentModal/AddDocumentModal.tsx

```diff
@@ -69,13 +77,17 @@ const AddDocumentModal = (props: Props) => {
         setDisabled(true);
       }
       
+      const isEmptyFile = fileSize !== 0;
+      if (!isEmptyFile)
```

Comment:
Why this variable is called 'isEmptyFile'? If 'fileSize' is bigger than 0, than the result of expression is true, which means that your file is not empty. So, change the variable name or rewrite the condition. Also, in this if statement it looks like you're checking if file is not empty, and if it's true, you're putting the notification that it's empty)
Also, please, specify the variable type.